decorator.rb

```diff
@@ -0,0 +1,13 @@
+require_relative 'nameable'
+class Decorator < Nameable
+  def initialize(nameable)
+    super()
+    @nameable = nameable
+  end
+
+  attr_accessor :nameable
```

Comment:
[OPTIONAL]

Kindly move your attribute accessor immediately after the class declaration to make your code clean